playbook.yml

```diff
@@ -16,23 +16,23 @@
           - rsync
           - wget
 
-    - name: Download charlesfleche keys
-      get_url: 
-        url: https://github.com/charlesfleche.keys
-        dest: .
+    - name: Setup charlesfleches keys
+      authorized_key:
+          user: debian
+          state: present
+          key: https://github.com/charlesfleche.keys 
+          path: .ssh/authorized_keys 
```

Comment:
Why is this `path` needed ? Does Ansible errors without it ? What is the error ?

Reply:
There were no errors, but for some unknowns reason, if i didn't specify the path, ansible wouldn't write the keys to the .ssh/authorized_keys file